### ISSUE TITLE
Restoring from background

### ISSUE THREAD

**Issue by l3d00m** [ccrama/Slide#101]
## Info
No idea what happened exactly
## Stacktrace
```java.lang.RuntimeException: Unable to destroy activity {me.ccrama.redditslide/me.ccrama.redditslide.Activities.CommentsScreen}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.util.ArrayList.get(int)' on a null object reference
at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3763)
at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3781)
at android.app.ActivityThread.access$1400(ActivityThread.java:151)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1363)
at android.os.Handler.disp
atchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5349)
at java.lang.reflect.Method.invoke(N
ative Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:908)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:703)Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.util.ArrayList.get(int)' on a null object reference
at me.ccrama.redditslide.Fragments.CommentPage.onCre
ateView(CommentPage.java:182)
at android.support.v4.app.Fragment.performCre
ateView(Fragment.java:1965)
at android.support.v4.app.FragmentManagerImpl.moveToSt
ate(FragmentManager.java:1078)
at android.support.v4.app.FragmentManagerImpl.moveToSt
ate(FragmentManager.java:1259)
at android.support.v4.app.FragmentManagerImpl.moveToSt
ate(FragmentManager.java:1241)
at android.support.v4.app.FragmentManagerImpl.disp
atchReallyStop(FragmentManager.java:2080)
at android.support.v4.app.FragmentController.disp
atchReallyStop(FragmentController.java:213)
at android.support.v4.app.FragmentActivity.onReallyStop(FragmentActivity.java:749)
at android.support.v4.app.FragmentActivity.doReallyStop(FragmentActivity.java:735)
at android.support.v4.app.FragmentActivity.onDestroy(FragmentActivity.java:324)
at android.support.v7.app.AppComp
atActivity.onDestroy(AppComp
atActivity.java:161)
at android.app.Activity.performDestroy(Activity.java:6199)
at android.app.Instrument
ation.callActivityOnDestroy(Instrument
ation.java:1140)
at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3750)... 10 more```
Reported via Slide Crash Capture

**Comment by ccrama** [comment]
Think I fixed with 4a11cea2144a2e016eb28503aaa0cb31337ef935

**Comment by l3d00m** [comment]
Thanks. I'll try tomorrow if it occurs again.
